ha/core/cluster/cluster_manager.py

```diff
@@ -180,9 +188,86 @@ def add_node(self, node):
         _output, _err, _rc = self._execute.run_cmd(const.PCS_STATUS)
         Log.info(f"Cluster status output after add node: {_output}, {_err}, {_rc}")
 
+    def get_nodes_status(self):
+        _output, _err, _rc = self._execute.run_cmd(const.PCS_STATUS_NODES, check_error=False)
```

Comment:
Can you paste example output of pcs status nodes.

Reply:
okay

Reply:
done